include/rdmini/ssa_direct.h

```diff
@@ -9,70 +9,84 @@
 
 namespace rdmini {
 
-// KeyType must be unsigned integral
-// ValueType must be floating point
+// KeyType              must be unsigned integral
+// ValueType            must be floating point
+// RealDistribution     must be a real distribution over an interval [a,b]
 
-template <typename KeyType,typename ValueType>
+template <typename KeyType,
+         typename ValueType> 
 struct ssa_direct {
     typedef KeyType key_type;
     typedef ValueType value_type;
+    typedef value_type& reference;
 
```

Comment:
This `reference` typedef doesn't appear to be used elsewhere — do we still need it?